app/javascript/shared/helpers/Validators.js

```diff
@@ -17,3 +17,7 @@ export const isValidPassword = value => {
   );
 };
 export const isNumber = value => /^\d+$/.test(value);
+export const isDomain = value => {
+  const domainRegex = /^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/;
```

Comment:
We can improve this regex, this will ensure we include international domains too.  See [regex101](https://regex101.com/r/qiiQIm/1)

```regexp
^([\p{L}0-9]+(-[\p{L}0-9]+)*\.)+[a-z]{2,}$
```

P.S. The \p{L} character class represents any letter from any language.

Reply:
@scmmishra I tried it with regex `/^([\p{L}0-9]+(-[\p{L}0-9]+)*\.)+[a-z]{2,}$/gm`, but the validation is not working correctly.

**Screen recording**
https://user-images.githubusercontent.com/64252451/210799813-525b4172-3bf0-4c93-a6de-f87ff704c184.mov

Reply:
This is a good suggestion. The suggested regex is not working as the flavour is set to PCRE2 instead of Javascript. Can you help out here  @scmmishra

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6166/commits/a02dc6ea51625b32fd30058a366be2bcbf8a5e73